docs/sources/examples/mongodb.md

```diff
@@ -133,19 +136,19 @@ as daemon process(es).
 
     # Basic way
     # Usage: docker run --name <name for container> -d <user-name>/<repository>
-    $ docker run --name mongo_instance_001 -d my/repo
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo
 
     # Dockerized MongoDB, lean and mean!
     # Usage: docker run --name <name for container> -d <user-name>/<repository> --noprealloc --smallfiles
-    $ docker run --name mongo_instance_001 -d my/repo --noprealloc --smallfiles
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo --noprealloc --smallfiles
 
     # Checking out the logs of a MongoDB container
     # Usage: docker logs <name for container>
     $ docker logs mongo_instance_001
 
     # Playing with MongoDB
-    # Usage: mongo --port <port you get from `docker ps`> 
-    $ mongo --port 12345
+    # Usage: mongo --port <port you get from `docker ps`>  --host <ip address from `boot2docker ip`>
+    $ mongo --port 27017 --host 192.168.59.103
```

Comment:
I know this was here before; Is this assuming that the `mongo` binary is installed on the users host here? If it is, perhaps we can rewrite the example to use a containerised mongo client. Using a `--link` to connect to the mongo daemon. This would circumvent the whole boot2docker / non-boot2docker IP address issue, and the port-binding won't be a problem as well.